services/tests/test_api_feedback.py

```diff
@@ -40,6 +40,24 @@ def test_cannot_get_feedback(self):
         rsp = self.client.get(url)
         self.assertEqual(NOT_FOUND, rsp.status_code, msg=rsp.content.decode('utf-8'))
 
+    def test_staff_sat_required_if_service_provided(self):
+        example = FeedbackFactory(
+            delivered=True,
+            staff_satisfaction=None,
+        )
+        self.try_to_create_one_like(example, expected_status=BAD_REQUEST)
+        example.staff_satisfaction = ''
+        self.try_to_create_one_like(example, expected_status=BAD_REQUEST)
+        example.staff_satisfaction = 3
+        self.try_to_create_one_like(example, expected_status=CREATED)
+
+    def test_staff_sat_not_required_if_service_not_provided(self):
+        example = FeedbackFactory(
+            delivered=False,
+            staff_satisfaction=None,
+        )
+        self.try_to_create_one_like(example)
```

Comment:
Minor nit that is might be a little more readable to explicitly pass the `expected_status=CREATED` here.